_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -11,17 +11,191 @@ redirect_from:
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td rowspan="4">Latency</td>
+        <td>p50</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+</table>
+
+Your results may vary based on cluster topology, hardware used, shard count, and merge settings. 
+{: .note }
+
+## Other considerations
+
+When using segment replication, consider the following:
+
+1. Enabling segment replication for an existing index requires [reindexing](https://github.com/opensearch-project/OpenSearch/issues/3685).
+1. Rolling upgrades currently are not supported. Full cluster restarts are required when upgrading indexes using segment replication. [Issue 3881](https://github.com/opensearch-project/OpenSearch/issues/3881).
+1. [Cross-cluster replication](https://github.com/opensearch-project/OpenSearch/issues/4090) currently does not use segment replication to copy between clusters.
+1. Increased network congestion on primary shards. [Issue - Optimize network bandwidth on primary shards](https://github.com/opensearch-project/OpenSearch/issues/4245).
+1. Integration with remote-backed storage as the source of replication is [currently unsupported](https://github.com/opensearch-project/OpenSearch/issues/4448). 
+1. Read-after-write guarantees: The `wait_until` refresh policy is not compatible with segment replication. If you use the `wait_until` refresh policy while ingesting documents, you'll get a response only after the primary node has refreshed and made those documents searchable. Replica shards will respond only after having written to their local translog. We are exploring other mechanisms for providing read-after-write guarantees. For more information, see the corresponding [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6046).  
+1. System indexes will continue to use document replication internally until read-after-write guarantees are available. In this case, document replication does not hinder the overall performance because there are few system indexes.
+
+## Next steps
 
-This is the first step in a series of features designed to decouple reads and writes in order to lower compute costs.
\ No newline at end of file
+1. Track [future enhancements on segment replication](https://github.com/orgs/opensearch-project/projects/99).
+1. [Segment replication blog post](https://github.com/opensearch-project/project-website/pull/1504/files).
```

Comment:
Is this the correct link?

Reply:
no, it will be updated once the blog goes live